_data/changelogs/component-footer.yml

```
@@ -2,6 +2,13 @@ title: Footer
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Restored layout-grid dependency in footer package and removed layout-grid styles from footer stylesheet.
+    summaryAdditional:  This update prevents visual regression in other components with layout-grid utility classes in their markup.
```

For clarity, I recommend we replace "layout-grid" references with either "usa-layout-grid" or "layout grid". 

```suggestion
    summary: Restored the `usa-layout-grid` dependency in the footer package and removed layout grid styles from the footer stylesheet. 
    summaryAdditional:  This update prevents visual regressions in footer and other components with layout grid utility classes in their markup.
```